action.yml

```diff
@@ -308,6 +313,11 @@ runs:
         Invoke-Expression -Command "$env:GITHUB_ACTION_PATH/venv/Scripts/Activate.ps1"
         pip install -r "$env:GITHUB_ACTION_PATH/requirements.txt"
         clang-tools -i ${{ inputs.version }} -b
+        $colorEnabled = "true","on","1"
+        if ( $colorEnabled.contains("${{ env.CPP_LINTER_COLOR }}") )
+        {
+            pip install -r "$GITHUB_ACTION_PATH/color-requirements.txt"
+        }
```

Comment:
1. When run in an unix environment, this patch include a `case` statement to satisfy this concern: https://github.com/cpp-linter/cpp-linter-action/blob/5f14ac81bb0f75f102ae444a81fc801d0fdb5959/action.yml#L272-L276
2. The `${{ env.CPP_LINTER_COLOR }}` syntax works because this action.yml file is used as GitHub actions composite action. Furthermore, this syntax env conforms to the environment specific to this action's context.